src/data/finset/lattice.lean

```diff
@@ -423,10 +423,10 @@ lemma sup_inf_distrib_right (s : finset ι) (f : ι → α) (a : α) :
   s.sup f ⊓ a = s.sup (λ i, f i ⊓ a) :=
 by { rw [_root_.inf_comm, s.sup_inf_distrib_left], simp_rw _root_.inf_comm }
 
-lemma disjoint_sup_right : disjoint a (s.sup f) ↔ ∀ i ∈ s, disjoint a (f i) :=
+protected lemma disjoint_sup_right : disjoint a (s.sup f) ↔ ∀ i ∈ s, disjoint a (f i) :=
```

Comment:
This change is already in master; did you merge an old version of master?

Reply:
Can you `git fetch origin` and `git merge origin/master` again, to be sure?

Reply:
Oh apparently I wanted to protect this in two different PRs: the present and #17961